runtime/src/main/java/org/corfudb/protocols/logprotocol/MultiObjectSMREntry.java

```diff
@@ -92,8 +91,11 @@ public void deserializeBuffer(ByteBuf b, CorfuRuntime rt) {
         for (int i = 0; i < numStreams; i++) {
             UUID streamId = new UUID(b.readLong(), b.readLong());
 
-            int updateLength = b.readInt();
-            byte[] streamUpdates = new byte[updateLength];
+            int start = b.readerIndex();
+            MultiSMREntry.seekToEnd(b);
```

Comment:
Can we add a minor comment on the stream payload structure (and that this is required because the payload size is not explicitly given)

Reply:
done.